config/redis.yml

```diff
@@ -14,6 +14,9 @@ development: &defaults
     namespace: user_identities
     each_ttl: 1800
   # DO NOT CHANGE ABOVE TTL
+  user_profile_attributes:
+    namespace: user-profile-attributes
+    each_ttl: 86400
```

Comment:
Could we still make this smaller? I'm thinking max 2-3 days? Is there any reason for it to be longer? Right now this is 60 days which is prob too long

Reply:
Oops, you're right, my math was wrong (`each_ttl` is indeed in seconds. So this is currently set to 24 hours, which seems right)